app/service_invite/rest.py

```diff
@@ -92,6 +96,35 @@ def update_invited_user(service_id, invited_user_id):
     return jsonify(data=invited_user_schema.dump(fetched)), 200
 
 
+@service_invite.route(
+    "/service/<service_id>/invite/<invited_user_id>/resend", methods=["POST"]
+)
+def resend_service_invite(service_id, invited_user_id):
+    """Resend an expired invite.
+
+    This resets the invited user's created date and status to make it a new invite, and
+    sends the new invite out to the user.
+
+    Note:
+        This ignores the POST data entirely.
+    """
+    fetched = get_invited_user_by_service_and_id(
+        service_id=service_id, invited_user_id=invited_user_id
+    )
+
+    fetched.created_at = datetime.utcnow()
+    fetched.status = INVITE_PENDING
+
+    current_data = {k: v for k, v in invited_user_schema.dump(fetched).items()}
+    update_dict = invited_user_schema.load(current_data)
+
+    save_invited_user(update_dict)
```

Comment:
If I'm reading this correctly based on the comment above and the code, this is finding the existing invite for a user regardless of its status, copying some of the details, and creating a new invite out of it, right?

If my understanding is correct, then this also relies on the job that automatically runs to expire invites two days older or more?  I just wanted to make sure that piece wasn't lost and we maintain that behavior so we still have properly expired invites.

Reply:
I didn't filter the invite based on status, true. I could do that as an additional filter, but I did make the assumnption that if someone calls this,, it was from the button/link ont he front-end to resend the invite, and would be expired.

But this does make that assumption, and I wasn't thinking about a bad-actor just being able to hack and use the endpoint to send a new invite directly. So... I can see that filtering would make sense.